vehicle-model-lifecycle/test/unit/test_download_vspec.py

```diff
@@ -137,10 +137,11 @@ def test_main__no_vehicle_signal_interface__adds_default_to_cache():
     with capture_stdout() as capture, mock_env():
         main(app_manifest)
 
-        expected_path = str(
-            Path(__file__).parent.parent.joinpath(
-                "vehicle-model-lifecycle", "vspec.json"
-            )
-        )
+        expected_path = str(os.path.join(get_project_cache_dir(), "vspec.json"))
+        expected_unit_path = str(os.path.join(get_project_cache_dir(), "units.yaml"))
         expected_cache_line = f"vspec_file_path={expected_path!r} >> VELOCITAS_CACHE\n"
-        assert capture.getvalue() == expected_cache_line
+        expected_unit_cache_line = (
+            f"unit_file_list={expected_unit_path!r} >> VELOCITAS_CACHE\n"
+        )
+        assert capture.getvalue().find(expected_cache_line)
+        assert capture.getvalue().find(expected_unit_cache_line)
```

Comment:
isn't this just checking if the lines are written to stdout? wouldn't it make more sense to check if it is really written to the cache?
`cache_data = get_cache_data()`

Reply:
yes, then the tests would've failed ;)

Reply:
but actually we can not test this here because cli writes the cache and we just write environment variable that tells the cli what to write. If I understood correctly @dennismeister93

Reply:
the other problem here is this check does not fail because find returns index and not true or false so I will change to check for ```!=-1``` because thats return if failed.

Reply:
if we can't test the cache here, than this test is useless IMO...